crates/bevy_tasks/src/task_pool.rs

```diff
@@ -50,13 +56,21 @@ impl TaskPoolBuilder {
         Self::default()
     }
 
-    /// Override the number of threads created for the pool. If unset, we default to the number
+    /// Override the number of threads created for the pool. If unset, it will default to the number
     /// of logical cores of the system
     pub fn num_threads(mut self, num_threads: usize) -> Self {
         self.num_threads = Some(num_threads);
         self
     }
 
+    /// Override the maximum number of blocking threads created for the pool. If unset, it will
```

Comment:
This and the above comment should talk about the difference between setting `num_threads` vs settings `max_blocking_threads`

Reply:
Done.